guides/tutorials/codelabs/kubernetes-source-to-prod/index.md

```diff
@@ -50,17 +50,21 @@ Follow one of the guides [here](http://kubernetes.io/docs/getting-started-guides
 
 ## Configuring Spinnaker
 
-We will be deploying Spinnaker to the same Kubernetes cluster it will be managing. To do so, follow the steps in [this guide](https://github.com/spinnaker/spinnaker/tree/master/experimental/kubernetes/simple), being sure to use [this section](https://github.com/spinnaker/spinnaker/tree/master/experimental/kubernetes/simple/#anything-else-except-for-ecr) to configure your registry.
+We will be deploying Spinnaker to the same Kubernetes cluster it will be managing. To do so, follow the installation [guide](/setup/install), being sure to enable & create a Kubernetes account, as well as a Docker Registry account with credentials and repositories pointing at those created above
 
 # 1. Create a Spinnaker Application
 
 Spinnaker applications are groups of resources managed by the underlying cloud provider, and are delineated by the naming convention `<app name>-`. Since Spinnaker and a few other Kubernetes-essential pods are already running in your cluster, your _Applications_ tab will look something like this:
 
-![Applications data and spin were created by Spinnaker, the rest were created by Kubernetes.](applications.png)
+Applications data and spin were created by Spinnaker, the rest were created by Kubernetes.
+
+![](applications.png)
 
 Under the _Actions_ dropdown select _Create Application_ and fill out the following dialog:
 
-![If you've followed the Source to Prod tutorial for the VM based providers, you'll remember that you needed to select "Consider only cloud provider health when executing tasks". Since Kubernetes is the sole health provider by definition, selecting this here is redundant, and unnecessary.](appfill.png)
+If you've followed the Source to Prod tutorial for the VM based providers, you'll remember that you needed to select "Consider only cloud provider health when executing tasks". Since Kubernetes is the sole health provider by definition, selecting this here is redundant, and unnecessary.
```

Comment:
@duftler these are the kinds of text/image combos that aren't rendered after the migration from readme